packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.module.scss

```diff
@@ -0,0 +1,11 @@
+:local {
+  .long-text-example {
```

Comment:
Since teams might look to copy this as their exact example, let's tighten up default style from 1em(14px) to 0.71428rem(10px) on the  basic `<p>` tag, which will make it look like default styling/standard spacer sizes.
```
p {
      margin: 0.71428rem 0;
}
```